packages/tools/client-debugger/client-debugger-view/src/RenderClientDebugger.ts

```diff
@@ -9,38 +9,21 @@ import { FluidClientDebuggers } from "./Debugger";
 
 /**
  * Renders Fluid client debug view by appending it to the provided DOM element.
- * Will fast return false if the input is `null`.
  *
  * @param targetElement - The HTML element takes the client debugger view.
  *
- * @returns `true` if the debug view was succesfully rendered, otherwise `false`.
+ * @returns A promise that resolves once the debugger view has been rendered for the first time.
+ * If rendering fails for any reason, the promise will be rejected.
  */
-export async function renderClientDebuggerView(
-	// eslint-disable-next-line @rushstack/no-new-null
-	targetElement: HTMLElement | null,
-): Promise<boolean> {
-	if (targetElement === null) {
-		console.log("Provided null targetElement.");
-		return false;
-	}
-
+export async function renderClientDebuggerView(targetElement: HTMLElement): Promise<void> {
```

Comment:
Calling this several times produces weird behavior, right? No need to address it here, thinking about follow-up work.

Reply:
Yes, this should just be called once for as long as the target element exists. I can add a note to that effect in the comment,